string[6] joint_names
float32[6] goal_position
float32[6] goal_velocity